packages/lesswrong/components/common/Header.tsx

```diff
@@ -328,11 +328,11 @@ const Header = ({standaloneNavigationPresent, sidebarHidden, toggleStandaloneNav
                 </NoSSR>
                 {!isEAForum && usersMenuNode}
                 {!currentUser && <UsersAccountMenu />}
-                {currentUser && <KarmaChangeNotifier
+                {currentUser && !currentUser.usernameUnset && <KarmaChangeNotifier
                   currentUser={currentUser}
                   className={(isEAForum && searchOpen) ? classes.hideXsDown : undefined}
                 />}
-                {currentUser && <NotificationsMenuButton
+                {currentUser && !currentUser.usernameUnset && <NotificationsMenuButton
```

Comment:
I also decided to hide these two icons on that page you land on when signing up, because they aren't useful until you finish signing up anyway so are just visually distracting and take up header space. This is what that page looks like now (I also this this page needs a makeover):

<img width="380" alt="Screen Shot 2023-07-05 at 11 26 52 PM" src="https://github.com/ForumMagnum/ForumMagnum/assets/9057804/e5a0e717-4411-4357-89e6-1568ef72d752">